plugin.video.embycon/resources/language/English/strings.po

```diff
@@ -0,0 +1,544 @@
+msgid ""
```

Comment:
can you please adopt the new language folder layout?

This should be:

`/resources/language/resource.language.en_gb/strings.po` 

see here: http://kodi.wiki/view/language_support